File: src/main/java/br/com/mecanopecas/mecanopecas/util/dtos/response/ClienteResponseDTO.java
package br.com.mecanopecas.mecanopecas.util.dtos.response;

import java.util.Date;
import java.util.List;

public record ClienteResponseDTO(Long id,
                                 String cpfCnpj,
                                 String nome,
                                 //List<OrcamentoResponseDTO> orcamentos,
                                 Date dataNascimento,
                                 boolean ativo ) { }

File: front-mecanopecas/src/dtos/response/clienteResponseDTO.ts
import { OrcamentoResponseDTO } from './orcamentoResponseDTO';

export interface ClienteResponseDTO {
    id: bigint;
    cpfCnpj: string;
    nome: string;
    orcamentos: OrcamentoResponseDTO[];
    dataNascimento: Date;
    ativo: boolean;
}
